API-GUIDELINE.md

```diff
@@ -0,0 +1,254 @@
+# API GUIDELINES
+
+## Overview
+As commercetools is 100% API First, it's important to have a well designed developer experience.
+Additionally we aim to achieve more automation and improve the review process of new changes in the APIs.
+
+With this in mind, we have created "API Guidelines" that aims to help developers to design APIs:
+
+ - simple
+ - consistent 
+ - self-explanatory
+ 
+The Keywords to have a good Guidelines are:
+ 
+ - Consistency => following the Guidelines
+ - Uniformity => checking in the code if there are other properties with similar name and adjust in base of it or improve it
+ - Explicability => explain with a few words the purpose of the property avoiding abbreviations (there are some [exceptions](#exceptions)).
```

Comment:
```suggestion
 - Explicability => explaining in a few words the purpose of the property avoiding abbreviations (there are some [exceptions](#exceptions)).
```

Just because other items start with -ing.